fre/frepp/configUpdates/pp.yaml

```diff
@@ -0,0 +1,225 @@
+## User defined edits
+configuration_paths: 
+  rose-suite: "/home/Dana.Singh/pp/ppyaml/fre-cli/fre/frepp/configUpdates/rose-suite-am5_c96L33_amip.conf" 
+  rose-remap: "/home/Dana.Singh/pp/ppyaml/fre-cli/fre/frepp/configUpdates/remap-rose-app.conf"
+  rose-regrid: "/home/Dana.Singh/pp/ppyaml/fre-cli/fre/frepp/configUpdates/regrid-rose-app.conf"
+
+analysisLen: &ANA_AMIP_LEN "10yr"
+chunk96: &PP_AMIP_CHUNK96 "1yr"
+analysisStart: &ANA_AMIP_START "1980"
+analysisStop: &ANA_AMIP_STOP "1988"
+defxyInterp: &defaultxyInterp "180,288"
+
+rose-suite:
+  refineDiag:
+    datastager_script: "$(includeDir)/refineDiag/refineDiag_data_stager_globalAve.csh"
+    atmoscmip6_script: "$(includeDir)/refineDiag/refineDiag_atmos_cmip6.csh"
+
+  settings:
+    history_segment: "P1Y"
+    pp_start: *ANA_AMIP_START
+    pp_stop: *ANA_AMIP_STOP
+    len: *ANA_AMIP_LEN
+    site: "ppan"
+    fre_analyis_home: #"/home/fms/local/opt/fre-analysis/test"
+    pp_chunk_a: "P5Y"
+    pp_chunk_b: "P10Y"
+    pp_grid_spec: "/archive/oar.gfdl.am5/model_gen5/inputs/c96_grid/c96_OM4_025_grid_No_mg_drag_v20160808.tar"
+
+  switches:
+    clean_work: True
+    do_mdtf: False
+    do_statics: True
+    do_timeavgs: True
+    do_refinediag: False
+    do_atmos_plevel_masking: True
+    do_preanalysis: False
+
+  directories:
+    history_dir: "/archive/c2b/am5/2022.01/c96L33_am5a0_cmip6Diag/gfdl.ncrc5-intel22-classic-prod-openmp/history"
+    pp_dir: "/archive/$USER/canopy/am5/c96L33_amip/pp"
+    ptmp: "/xtmp/$USER/ptmp"
+    refine_diag:
+    preanalysis:
+    history_refined: "/archive/$USER/canopy/am5/c96L33_amip/history_refineDiag"
+    analysis: #"/nbhome/$USER/canopy/analysis/am5_c96L33_amip"
+
+  metadata:
+    experiment: "am5_c96L33_amip"
+    platform: "gfdl.ncrc5-intel22-classic"
+    target: "prod-openmp"
+
+#regrid-xy and remap-pp-components information
+regrid-remap-info:
```

Comment:
Yes! The `components` part down is perfect as is, clearly analogous to a Bronx XML and I think users will be able to understand it.

Can the `regrid-remap-info` header be removed?

Reply:
Yes, it can. I put that in there for more organization but I'm sure it would work with just `components` as the header here.